src/components/sections/action-item.tsx

```diff
@@ -13,20 +13,22 @@ export type ActionItemProps = SectionItem<{
   onPress?(checked: boolean): void;
   checked?: boolean;
   mode?: ActionModes;
+  accessibility?: AccessibilityProps;
 }>;
 export default function ActionItem({
   text,
   onPress,
   mode = 'check',
   checked = false,
+  accessibility,
   ...props
 }: ActionItemProps) {
   const {contentContainer, topContainer} = useSectionItem(props);
   const style = useSectionStyle();
 
   if (mode === 'toggle') {
     return (
-      <View style={[style.baseItem, style.spaceBetween, topContainer]}>
+      <View style={[style.spaceBetween, topContainer]} {...accessibility}>
```

Comment:
Generally, corresponding label should respond to click action by toggling the switch, 
and I wonder if we will make the function more understandable by doing so as well. If the entire containing element has role switch, and an onPress action. 
Right now, for the usage in favorites toggler, talkback reads "Vis favoritter" (no action provided), and then by navigating to the next item, the switch is read out so fast that I cant interpret what is said.  (only the following hint about how to interact, which I think may be Android specific?) Probably default behaviour, but I'm not sure if it's understandable enough in our case

Reply:
> Generally, corresponding label should respond to click action by toggling the switch,

Maybe there are different practices on Android / iOS here? I know checkboxes and labels are a thing on the Web but typically iOS pattern is to _not_ have the label (entire line) be pressable to toggle Switches.

Reply:
Should we possibly do text as inaccessible and adding an accessible label equal to the text on the Switch itself?

Reply:
Okay, seems like clickable label for switch is an Android-only pattern then.

Reply:
Your suggestion on labeling the switch: Yes, I think that would make more sense =)

Reply:
Hm. Then we should probably do so on android. I can look into it

Reply:
I did some accessibility improvements. But not the clickable thing. I think we should work out a good way to do both patterns on the platforms, but tempted to say we should do that as a separate issue as it affects many other things aswell